api/NodeList.json

```diff
@@ -268,10 +268,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
Wait, no, this is not plausible, the `length` attribute must have been around much earlier than this.

http://mdn-bcd-collector.appspot.com/tests/api/NodeList tests the prototype only, is Safari 10 when attributes were moved to prototypes?

Reply:
Ack, my bad, I didn't mean to include this change...  Reverted!